src/backend/lang_c/c_simple_ast.rs

```diff
@@ -438,11 +484,21 @@ impl<'a> CASTConverter<'a> {
     /// Entry point of Simple-C-AST to C-AST conversion.
     pub fn to_c_ast(&mut self) -> CAST {
         let mut c_ast = CAST::new(&self.ast.fname);
-        for var in self.ast.vars.iter() {
-            if let Some(&SimpleCASTNode::Value(ValueNode::Variable(ref ty_opt, ref var_name))) = self.ast.ast.node_weight(*var) {
+        // XXX
```

Comment:
Same thing as @chinmaydd said, please write at least a line as to what this XXX is, it may be clear to you, but anyone else reading this code will not understand what this is for or what's missing.